functions/transformations/assert_equals.go

```diff
@@ -0,0 +1,239 @@
+package transformations
+
+import (
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/plan"
+	"github.com/influxdata/flux/semantic"
+)
+
+const AssertEqualsKind = "assertEquals"
+
+type AssertEqualsOpSpec struct {
+	Name string `json:"name"`
+}
+
+func (s *AssertEqualsOpSpec) Kind() flux.OperationKind {
+	return AssertEqualsKind
+}
+
+func init() {
+	assertEqualsSignature := semantic.FunctionPolySignature{
+		Parameters: map[string]semantic.PolyType{
+			"name": semantic.String,
+			"got":  flux.TableObjectType,
```

Comment:
The examples show that `got` is the pipe forward argument, can you update the signature here for that?

Should we rename `got` to `tables` to make it the same as other chaining functions?

Reply:
I think it will be common enough to write a test `assertEquals(got: s1, want: s2)`  that `got` is better.  If it's a big issue, we can remove the pipe-forwarding from the spec.